pages/index.js

```diff
@@ -7,8 +7,8 @@ const Home = (props) => {
 
     return (
         <JssProvider id={{ minify: true }}>
-        <Layout toggleDarkMode={props.toggleDarkMode} connectUser={props.connectUser} provider={props.provider} user={props.user}>
-            <SignUp user={props.user} idx={props.idx} identity={props.identity} setUserData={props.setUserData} setUser={props.setUser} />
```

Comment:
Add the `setUser`. Needed to fix the signUp modal bug.

Reply:
The modal issue isnt there now. Check with social login it was working fine for me though.